### PR TITLE
OPSEXP-1928 Automate rpm Requires of the imagemagick packages

### PR DESCRIPTION
**Pull request by slohe1**
OPSEXP-1928

### CODE REVIEW

**Review by alxgomz — CHANGES_REQUESTED** [review]
:) nice try
Can you instead investigate a method that's not bound to specific version. We want to avoid having to maintain such if statements.

**Review by gionn — COMMENTED** [review]
don't want to add more entropy here, but I would:

* have sh fragments for each version instead of rpm_cap**.txt (e.g. override-7.0.10-59.sh)
* in the entrypoint.sh, add a `source override-$VERSION.sh` to execute that override before the `./configure`

File: .github/actions/rockylinux-build/entrypoint.sh [hunk 1]
```diff
@@ -16,10 +16,10 @@ dnf install -y epel-release git make wget 'dnf-command(config-manager)' && \
 git clone --depth 1 -b $1 https://github.com/ImageMagick/ImageMagick.git
 
 # Generate updated .src.rpm
-cd ImageMagick && \
+ cd ImageMagick && \
```
Comment (alxgomz) [inline .github/actions/rockylinux-build/entrypoint.sh:19]: please pay attention to formatting

```suggestion
cd ImageMagick && \
```

**Review by alxgomz — DISMISSED** [review]
small formatting issue

**Review by alxgomz — APPROVED** [review]

**Pull request merged**